### DISCUSSION TITLE
Blank screen after editing index.html file to add new parameters / options

### DISCUSSION THREAD

**Discussion by adeniyisikirat3** [EmulatorJS/EmulatorJS#467] [Q&A]
reCalling my previous suggestion here: https://github.com/EmulatorJS/EmulatorJS/discussions/464 

It may help to make provision for user to be able to set other options or parameters | I'm refering to the ones this page -> https://emulatorjs.org/docs/Options.html

For  the user to be able to include all or any of those options / parameters right here on the https://emulatorjs.org/editor.html  page. That is when creating.
,

This is Because for some reason anytime I try to change the index.html file to add the options / parameters from here: https://emulatorjs.org/docs/Options.html  any time I try to add some of those parameters it gives this blank screen error (see screenshot) -> : 
https://postimg.cc/fkNh1t5B

However if I don't touch the index.html page everything seemed to be working fine. I think this happens only when I try to set some of those parameters. 🤔

**Comment by ethanaobrien** [comment]
Chances are, there is something wrong with your syntax (you're imputing the values wrong). This may be done
